package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "bot",
+  "version": "1.0.0",
+  "description": "",
+  "main": "bot.js",
+  "dependencies": {
+    "mineflayer": "1.7.5",
+    "mineflayer-blockfinder": "0.0.5",
+    "mineflayer-navigate": "0.0.9"
+  },
+  "devDependencies": {
+    "flow-bin": "^0.33.0",
+    "flow-remove-types": "^1.0.4"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "flow; test $? -eq 0 -o $? -eq 2; flow-node"
```

Comment:
The windows example uses a different script here: https://flowtype.org/blog/2016/08/01/Windows-Support.html

"flow_check": "flow check || exit 0"

Not sure if it does the same thing?